### ISSUE TITLE
VCL issue

### ISSUE THREAD

**Issue by brentaligent** [aligent/Phoenix_VarnishCache#3]
With the finding of the issue with json responses being thought of as a static resource. It would be best to change the vcl fetch if test for 'text/html' || 'text/xml' || 'application/json' to instead define the static types either on mime type or url regex for path/extension so that if other formats are served such as json having mime type 'text/javascript' then it will not be considered a static resource.

Basically inverting the if statement so that static types are defined rather than dynamic ones.
